docs/topics/what-is-the-toolkit.adoc

```diff
@@ -14,19 +14,19 @@
 [discrete]
 == What is the {ProductName}?
 
-The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration and modernization of Java applications.
-
-{ProductShortName} examines application artifacts, including project source directories and application archives, and then produces an HTML report highlighting areas needing changes. {ProductShortName} supports many migration paths including the following examples:
-
-* Upgrading to the latest release of Red Hat JBoss Enterprise Application Platform
-* Migrating from Oracle WebLogic or IBM WebSphere Application Server to Red Hat JBoss Enterprise Application Platform
-* Containerizing applications and making them cloud-ready
-* Migrating from Java Spring Boot to Quarkus
-* Updating from Oracle JDK to OpenJDK
-* Upgrading from OpenJDK 8 to OpenJDK 11
-* Upgrading from OpenJDK11 to OpenJDK 17
-* Migrating EAP Java applicatons to Azure
-* Migrating Spring Boot Java applications to Azure
+The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies migrating and modernizing Java applications.
+
+{ProductShortName} examines application artifacts, including project source directories and application archives. It then produces an HTML report highlighting areas that need changes. {ProductShortName} supports many migration paths including the following examples:
+
+* Upgrade to the latest release of Red Hat JBoss Enterprise Application Platform

```

Comment:
@HagayVider1 
I would keep gerunds here. These are migration paths - you can hardly call a path "Migrate from Oracle...".  If you have something against gerunds, change them to nouns - migration, upgrade, containerization.